tests/backup/backup_test.go

```diff
@@ -7297,3 +7298,154 @@ var _ = Describe("{ScheduleBackupCreationAllNS}", func() {
 		DeleteCloudAccounts(backupLocationMap, cloudAccountName, cloudCredUID, ctx)
 	})
 })
+
+var _ = Describe("{CustomResourceRestore}", func() {
+	var (
+		contexts           []*scheduler.Context
+		appContexts        []*scheduler.Context
+		backupLocationUID  string
+		cloudCredUID       string
+		bkpNamespaces      []string
+		clusterUid         string
+		clusterStatus      api.ClusterInfo_StatusInfo_Status
+		backupName         string
+		credName           string
+		cloudCredUidList   []string
+		backupLocationName string
+		deploymentName     string
+		restoreName        string
+		backupNames        []string
+		restoreNames       []string
+	)
+	labelSelectors := make(map[string]string)
+	namespaceMapping := make(map[string]string)
+	newBackupLocationMap := make(map[string]string)
+	backupNamespaceMap := make(map[string]string)
+	deploymentBackupMap := make(map[string]string)
+	bkpNamespaces = make([]string, 0)
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("CustomResourceRestore", "Create custom resource restore", nil, 58041)
+		log.InfoD("Deploy applications")
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
+			appContexts = ScheduleApplications(taskName)
+			contexts = append(contexts, appContexts...)
+			for _, ctx := range appContexts {
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Create custom resource restore", func() {
+		providers := getProviders()
+		Step("Validate applications", func() {
+			log.Infof("Validate applications")
```

Comment:
Change this to `log.InfoD`

Reply:
Done

Reply:
I dont see this. It still shows Infof

Reply:
There was some glitch earlier, I've corrected it can you please confirm.